### PR TITLE
react/week1

### PR DESCRIPTION
**Pull request by semih1239**

### CODE REVIEW

File: react/week1/todo-list/src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import reportWebVitals from './reportWebVitals';
+import getTodos from './getTodos'
+
+const todos = getTodos()
+
+function Render(){
+  return (
+    <div>
+      {todos.map((todo) => {
+        return <Todos description={todo.description} deadline={todo.deadline} />
```
Comment (rufaidaju) [inline react/week1/todo-list/src/index.js]: Here the console is showing a warning that says Each child in a list should have a unique "key" prop.
you can avoid the problem by providing a unique key that has the value of todo.id

**Comment by rufaidaju** [comment]
Hi Semih
Good job with the homework.
I have small things to mention:
Name your components (file)
To figure which component has a bug, it’s important to always give your component a name. I would name App.js to be
Todo.js
It is also better especially when you have a big project, it helps you to organize your project and usually when you have many components we add them to one folder called components.
Good luck :)

File: react/week1/todo-list/src/index.js (hunk 1 above)
Comment (semih1239) [reply react/week1/todo-list/src/index.js]: Thanks I will update my codes 👍

**Comment by semih1239** [comment]
> Hi Semih
> Good job with the homework.
> I have small things to mention:
> Name your components (file)
> To figure which component has a bug, it’s important to always give your component a name. I would name App.js to be
> Todo.js
> It is also better especially when you have a big project, it helps you to organize your project and usually when you have many components we add them to one folder called components.
> Good luck :)

Thank you very much for your feedback. Sorry for late answer. I will update my codes 👍

**Pull request merged**